roles/pulibrary.sneakers_worker/molecule/default/molecule.yml-e

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
I think that this might have been committed in error using a `sed -e` invocation in a macOS environment.

Reply:
indeed it is. my muscle memory doesn't recall using `sed` howeever. It looks like it was generated by `invoke molecularize pulibrary.sneakers_workers`